### ISSUE TITLE
 Sidebar layout

### ISSUE THREAD

**Issue by GOTFire** [mit-cml/appinventor-sources#1488]
I use Makeroid layout as sidebar feature its really usefull for proffesional desing.There is too many apps use sidebar on the market and app inventor dont have built in sidebar option for this.Is it posible add using layouts as sidebar? I know its already integrated to Makeroid but its really usefull for all.

This option can work like this
1.We can select layout for sidebar.
We can use layouts in the screen or 
2.There is a Special panel for desing layouts as sidebar

I think 2. Option make it easy for users.

And if There is a option set sidebar for all or selected screens its will be good

**Comment by kwankiu** [comment]
I think what you mean would be the Android Navigation Drawer which is currently available in Kodular, Thunkable X, Android Studio, Sketchware, etc.

**Comment by kwankiu** [comment]
Maybe this could help ?
https://community.thunkable.com/t/free-extension-sidebar-navigation-update-12-04-2018/5376

**Comment by GOTFire** [comment]
Nope its not work correctly  and we cant use own desings

**Comment by GOTFire** [comment]
I already using Kodular thanks for it.I think app inventor 2 need have a built in option for this.

**Comment by AwaiisLaTif** [comment]
You can use Android Navigation Drawer for developing a sidebar. As I understood from above, you can use the Android Navigation Drawer on multiple screens and customize its design on each screen. It is an easy and good option for this one. much professional use this for the sidebar.

**Comment by GOTFire** [comment]
Yes but i vote for layout option because it is more  customizable.We can desing every think.Same as layout
